cypress/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+export const getTestSelector = (selectorId: string) => `[data-cy=${selectorId}]`
+
+// 1. Go to featureFlags/index.tsx
+// 2. Find the FeatureFlag enum
+// 3. Copy the string that correspond with the flag you want to enable and pass it here
+export const selectFeatureFlag = (featureFlag: string) => {
+  cy.get(getTestSelector('menu')).click()
+  cy.get(getTestSelector('feature-flags')).click()
+  cy.get(`#${featureFlag}`).select('enabled')
```

Comment:
This may not work, because loading a window clears the local storage (see cypress/support/e2e.ts?L=37).

Could this instead be added to the cypress/support/e2e.ts file, as part of the `visit` command?
Note that you can augment the `VisitOptions` type in that file, in L19 (as is done for `serviceWorker` tests).

Reply:
it does persist and this reload is consistent with the current logic for enabling widgets.  I think this is okay as is